package.json

```diff
@@ -2,34 +2,37 @@
   "name": "react-lifecycles-compat",
   "version": "1.1.0",
   "description": "Backwards compatibility polyfill for React class components",
-  "main": "index.js",
+  "main": "react-lifecycles-compat.cjs.js",
+  "module": "react-lifecycles-compat.es.js",
   "license": "MIT",
   "repository": "reactjs/react-lifecycles-compat",
   "scripts": {
-    "test": "jest test.js",
-    "build:dev": "NODE_ENV=development webpack",
-    "build:prod": "NODE_ENV=production webpack",
-    "build": "npm run build:dev && npm run build:prod",
+    "build": "rollup -c",
     "install:dependencies": "node install.js",
-    "postinstall": "npm run install:dependencies",
     "lint": "eslint index.js",
+    "postinstall": "npm run install:dependencies",
```

Comment:
Whoops. This change (from PR #4) actually breaks the package, because it tries to run for people NPM installing the package as well.

Reply:
Other than this, the changes seem good (based on a quick local test).

Reply:
Damn, totally forgot this might run for NPM installs too. `prepare` should do the trick for this though.
> prepare: Run both BEFORE the package is packed and published, and on local npm install without any arguments (See below). This is run AFTER prepublish, but BEFORE prepublishOnly.

Reply:
Yeah, I forget that one too.